src/components/VirtualKeyInfo.vue

```diff
@@ -0,0 +1,41 @@
+<script setup lang="ts">
+import { formatHertz, formatExponential, formatCents } from '@/utils'
+
+const props = defineProps<{
+  cent: number
```

Comment:
Should be `cents` instead. A single cent only refers to a ratio of 2^(1/1200).
Missed this on v3, so I'll fix it there myself.